roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -48,24 +90,28 @@
     path: "{{ drupal_deploy_dir }}/content/sync"
     owner: "{{ webserver_app_user }}"
     group: "{{ webserver_app_user }}"
+  when: webserver_app_do_content_sync_config or webserver_app_do_original_trusted_host_and_more
 
 - name: Import features
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y fim islandora_core_feature,islandora_defaults"
-    
+  when: webserver_app_feature_import_base
+
 - name: Import search feature
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y fim islandora_search"
-  when: islandora_profile == "standard"
+  when: webserver_app_feature_import_search
 
 # Set Matomo Settings
 - name: Set Matomo site id.
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y config-set matomo.settings site_id 1"
-  register: set_search_api_config
-  changed_when: "'Do you want to update site_id' in set_search_api_config.stdout"
+  register: set_matomo_site_id_config
+  changed_when: "'Do you want to update site_id' in set_matomo_site_id_config.stdout"
+  when: webserver_app_configure_matomo
 
-- name: Set Matmo sever url.
+- name: Set Matomo server url.
   command: "{{ drush_path }} --root {{ drupal_core_path }} -y config-set matomo.settings url_http http://localhost:8000/matomo/"
-  register: set_search_api_config
-  changed_when: "'Do you want to update site_id' in set_search_api_config.stdout"
+  register: set_matomo_server_url_config
+  changed_when: "'Do you want to update site_id' in set_matomo_server_url_config.stdout"
+  when: webserver_app_configure_matomo
```

Comment:
The `register`/`changed_when` things touching `set_search_api_config` seems like something of copypasta error, as we're not dealing with "search API" here... so adjusted things in a way that seems more consistent?

... it's... _possible_ that the second will have to have the string it's searching for changed... instead of `site_id`, to be `url_http` or something of the like.

Reply:
> ... it's... _possible_ that the second will have to have the string it's searching for changed... instead of `site_id`, to be `url_http` or something of the like.

Did indeed appear to be the case. Running the command manually: 
![Screen Shot 2022-09-13 at 12 10 52 PM](https://user-images.githubusercontent.com/607975/189938819-25d2a96a-30eb-4b60-836c-efb47b488cae.png)

Addressed in https://github.com/Islandora-Devops/islandora-playbook/pull/226/commits/dd550679f02cef9a3bc1961f2efc97d1981033fe